### PR TITLE
net-im/corebird: Version bump to 1.7.4

### PR DESCRIPTION
**Pull request by moschlar**
Closes: https://bugs.gentoo.org/529598
Bug: https://bugs.gentoo.org/636994

### CODE REVIEW

File: net-im/corebird/corebird-1.7.4.ebuild [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		media-plugins/gst-plugins-hls:1.0
+		media-plugins/gst-plugins-libav:1.0
+		media-plugins/gst-plugins-meta:1.0[X]
+		media-plugins/gst-plugins-soup:1.0 )
+	spellcheck? ( >=app-text/gspell-1.0[vala] )
+	>=net-libs/libsoup-2.42.3.1
+	>=x11-libs/gtk+-3.20:3"
+DEPEND="${RDEPEND}
+	$(vala_depend)
+	>=dev-util/intltool-0.40
```
Comment (monsieurp) [inline net-im/corebird/corebird-1.7.4.ebuild]: This version of intltool no longer exists in the main repository. Please update this line to `dev-util/intltool`.

File: net-im/corebird/corebird-1.7.4.ebuild [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+HOMEPAGE="https://corebird.baedert.org/"
+SRC_URI="https://github.com/baedert/corebird/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="debug gstreamer spellcheck"
+
+RDEPEND="dev-db/sqlite:3
+	>=dev-libs/glib-2.44:2
```
Comment (monsieurp) [inline net-im/corebird/corebird-1.7.4.ebuild]: This line can be updated to `dev-libs/glib:2`.

File: net-im/corebird/corebird-1.7.4.ebuild [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+	dev-libs/json-glib
+	gstreamer? ( >=media-libs/gst-plugins-bad-1.6:1.0[X,gtk]
+		media-libs/gst-plugins-base:1.0[X]
+		media-libs/gst-plugins-good:1.0
+		media-plugins/gst-plugins-hls:1.0
+		media-plugins/gst-plugins-libav:1.0
+		media-plugins/gst-plugins-meta:1.0[X]
+		media-plugins/gst-plugins-soup:1.0 )
+	spellcheck? ( >=app-text/gspell-1.0[vala] )
+	>=net-libs/libsoup-2.42.3.1
```
Comment (monsieurp) [inline net-im/corebird/corebird-1.7.4.ebuild]: This version no longer exists as well and the latest version available is at least 2.56.1. Please update this line to `net-libs/libsoup` then.

**Comment by moschlar** [comment]
I've added a "fix" for https://bugs.gentoo.org/636994 by patching out the execution of the one test that would perform a download operation.

**Pull request closed without merging**